### PR TITLE
Isort time

### PR DESCRIPTION
**Pull request by nstarman**
### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Do the proposed changes actually accomplish desired goals?
- [x] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [x] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [x] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [x] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [x] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [x] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [x] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [x] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [x] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

**Review by taldcroft — APPROVED** [review]
Looks good.

**Comment by taldcroft** [comment]
@nstarman  - The CI/Python 3.9 cancelled after 360m. This needs to get restarted and pass then this is good for merging.

File: astropy/time/config.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from astropy import config as _config
+
+
+class Conf(_config.ConfigNamespace):  # noqa
```
Comment (pllim) [inline astropy/time/config.py:6]: This is out of scope and probably an API change. Does not look related to isort.

Comment (nstarman) [reply astropy/time/config.py:6]: Can delete.
I don't thinks it's an API change for the user since the all the contents of ``config.py`` are dumped into ``__init__``. The only thing that is removed from ``__init__`` is the private variable ``_config``.

Comment (nstarman) [reply astropy/time/config.py:6]: I put this in here because then isort can sort more happily without ``E403`` errors related to splitting imports.

Comment (pllim) [reply astropy/time/config.py:6]: But you also didn't mention you moved it. I wouldn't have known without looking at the diff. I would prefer this be a separate PR from isort. Thanks!

Comment (pllim) [reply astropy/time/config.py:6]: Re: E403 -- Can isort ignore that for now? Or does that block this work?

Comment (nstarman) [reply astropy/time/config.py]: Moved to a subsequent PR.

Comment (nstarman) [reply astropy/time/config.py]: It doesn't block. I have to turn off `isort` for those imports anyway because they are circularly dependent.

File: astropy/time/setup_package.py [hunk 2]
```diff
@@ -3,9 +3,9 @@
 # Copied from astropy/convolution/setup_package.py
 
 import os
-from setuptools import Extension
```
Comment (pllim) [inline astropy/time/setup_package.py:6]: Same comment as https://github.com/astropy/astropy/pull/13699#pullrequestreview-1112310333 😅

Comment (nstarman) [reply astropy/time/setup_package.py:6]: Yes. Darn.

Comment (nstarman) [reply astropy/time/setup_package.py:6]: Either this PR waits for #13699, or we merge now and let isort fix here and there at some later merge. The next time someone pushes the result of ``isort .`` this will be cleaned up.

Comment (pllim) [reply astropy/time/setup_package.py:6]: Let's defer this to https://github.com/astropy/astropy/issues/13718 . Thanks!

**Review by pllim — APPROVED** [review]
Thanks!
